test/controllers/stories_controller_test.rb

```diff
@@ -317,4 +317,35 @@ def setup
 
     assert_response :unauthorized
   end
+
+  test "should create with story_points" do
```

Comment:
Troque o nome do teste por:
`test "should create a story with story points" `

Reply:
Serão necessários, por consequência, testes para quando um projeto não decidir pontuar suas histórias.

Reply:
Saquei, tipo um booleano "is_scoring" dentro de projeto né??

Reply:
Algo do tipo. Com base nessa variável deve-se ver se a pontuação vai ser OBRIGATÓRIA ou não. Isso é importante: num projeto que decidiu PONTUAR suas histórias, o atributo de story points deve ser obrigatório.

Reply:
Pronto, @MatheusRich ! Acho que atendi as suas solicitações.